scripts/utils.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+# To use these utilities you can source this file in your script
```

Comment:
I think we should add a note about contributing changes back to this repo (https://github.com/DataBiosphere/terra-landing-zone-service/pull/421#discussion_r1524015977).